minidump-processor/json-schema.md

```diff
@@ -219,6 +219,16 @@ anyway.
             | "frame_pointer"  // Used frame pointers to unwind (often reliable)
             | "scan",          // Searched the callee's stack memory (SKETCHY!)
 
+          // The values the general purpose registers contained (frame 0 only!).
```

Comment:
I think instead of "frame 0 only" I would say something like "In the default config, this field will only show up in frame 0 of the crashing_thread copy of this backtrace. In the future this may be more configurable and appear in different frames."